### PR TITLE
Add a Call for Hosts

### PR DESCRIPTION
**Pull request by rayegun**
@ranjanan @vchuravy @kshyatt @aviks (and any other committee members, I just ping'd long time members) please take a look at this. This would be the first time we've done it this way I think, but it seems reasonable to me.

### CODE REVIEW

File: config.md [hunk 1]
```diff
@@ -158,14 +158,14 @@ configuration = Dict(
             "Store" => "/2024/store",
             "Sponsor" => "/2024/sponsor",
             "Venue" => "/2024/travel",
-            "Childcare" => "/2024/dei",
```
Comment (ranjanan) [inline config.md:161]: any reason to remove a link to the DEI page?

Comment (ranjanan) [reply config.md:161]: Otherwise LGTM. I will post a link to this page on Discourse.

Comment (rayegun) [reply config.md:161]: We can't add people to childcare afaik, and there is still a link to it under Guidelines. @skygering is that reasonable to you? Happy to do something different

Comment (pabloferz) [reply config.md:161]: We also have this in the main page:
image
so I think it should be safe to place this in submenu.

**Comment by skygering** [comment]
I thought we were leaving up childcare signups for a little bit. But also we have 10 kids signed up which is a good portion of the 15 (and that doesn't include the PyData kids) so I think we are good to remove the link :D

**Comment by ranjanan** [comment]
I am just merging this so I can make the discourse post and start the conversation.

**Pull request merged**